### PR TITLE
EDU-1196 Support trust logos admin setup

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1196

Screen Shot 2023-03-06 at 19 15 42
Screen Shot 2023-03-06 at 19 17 59

### CODE REVIEW

File: src/components/admin/admin-settings-tab.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@ adminSettingsTab:
   homepagePresentationHeader:
     en: Homepage presentation
     de: Homepage-Präsentation
+  homepageTrustLogosHeader:
+    en: Homepage trust logos
+    de: Homepage Vertrauens-Logos
```
Comment (ahelmberger) [inline src/components/admin/admin-settings-tab.yml:28]: Let's also connect "Homepage" with "Vertrauens" by a hyphen.

File: src/components/admin/homepage-trust-logos-settings.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+homepageTrustLogosSettings:
+  institution:
+    en: Institution
+    de: Institution
+  logoUrl:
+    en: Logo URL
+    de: Logo-URL
+  institutionUrl:
+    en: Institution URL
+    de: Institution-URL
```
Comment (ahelmberger) [inline src/components/admin/homepage-trust-logos-settings.yml:10]: Institutions-URL

File: src/components/admin/admin-settings-tab.yml [hunk 3]
```diff
@@ -23,6 +23,12 @@ adminSettingsTab:
...
+  homepageTrustLogosInfo:
+    en: The logos of the trusted institutions are displayed in an extra footer on the homepage. Please provide SVGs with transparent background and links to the home pages of each institution.
+    de: The logos of the trusted institutions are displayed in an extra footer on the homepage. Please provide SVGs with transparent background and links to the home pages of each institution.
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/components/admin/admin-settings-tab.yml:31]: 👍 thanks!

File: src/components/admin/homepage-trust-logos-settings.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/admin/homepage-trust-logos-settings.yml:10]: 's changed, thanks

File: src/components/admin/admin-settings-tab.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/admin/admin-settings-tab.yml:28]: Connected!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**